packages/db/fauna/resources/Function/findMetrics.js

```diff
@@ -0,0 +1,82 @@
+import fauna from 'faunadb'
+
+const {
+  Function,
+  CreateFunction,
+  Query,
+  Lambda,
+  Count,
+  Exists,
+  If,
+  Update,
+  Sum,
+  Match,
+  Index,
+  Map,
+  Get,
+  Var,
+  Select,
+  Documents,
+  Collection,
+  Paginate
+} = fauna
+
+const name = 'findMetrics'
+const body = Query(
+  Lambda(
+    [],
+    {
+      data: {
+        usersTotal: Count(Documents(Collection("User"))),
+        uploadsTotal: Count(Documents(Collection("Upload"))),
+        contentTotal: Count(Documents(Collection("Content"))),
+        contentTotalBytes: Sum(Match(Index("content_sizes"))),
+        dealsActiveTotal: Count(Match(
+          Index("deals_by_status"),
+          'Active'
+        )),
+        dealsQueuedTotal: Count(Match(
+          Index("deals_by_status"),
+          'Queued'
+        )),
+        pinsTotal: Count(Documents(Collection("Pin"))),
+        pinsTotalBytes: Sum(
+          Select('data', Map(
+            Paginate(
```

Comment:
Using this to convert set to array, couldn't find an alternative 🤷🏼

Reply:
This will give you just 1 page of data - 64 items by default. You can increase the page size up to 100K but after that we'll need to page through.